src/documents/TextToSpeechAPI.md

```diff
@@ -0,0 +1,580 @@
+                    </dd>
+                </dl>
+            </td>
+        </tr>
+        <tr>
+            <th>Example</th>
+            <td>
+<pre>
+<code>
+fluid.queueSpeech("Hello world", false, {
+    volume: 0.5
+});
+</code>
+</pre>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### `cancel` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Method</th>
+            <td>
+                <code>textToSpeech.cancel();</code>
+            </td>
+        </tr>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>cancel</code> method allows you to remove all texts from the queue. If a text is being spoken, speaking ceases immediately.
+            </td>
+        </tr>
+        <tr>
+            <th>Parameters</th>
+            <td>none</td>
+        </tr>
+    </tbody>
+</table>
+
+### `pause` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Method</th>
+            <td>
+                <code>textToSpeech.pause();</code>
+            </td>
+        </tr>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>pause</code> method allows you to immediately pause any texts that are being spoken..
+            </td>
+        </tr>
+        <tr>
+            <th>Parameters</th>
+            <td>none</td>
+        </tr>
+    </tbody>
+</table>
+
+### `resume` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Method</th>
+            <td>
+                <code>textToSpeech.resume();</code>
+            </td>
+        </tr>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>resume</code> method allows you to resume speaking a text that was previously paused.
+            </td>
+        </tr>
+        <tr>
+            <th>Parameters</th>
+            <td>none</td>
+        </tr>
+    </tbody>
+</table>
+
+### `getVoices` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Method</th>
+            <td>
+                <code>textToSpeech.getVoices();</code>
+            </td>
+        </tr>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>getVoices</code> method allows you to get a list of all the voices that are supported by the browser.
+            </td>
+        </tr>
+        <tr>
+            <th>Parameters</th>
+            <td>none</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Supported Events ##
+
+_**Note:** If needed, please read the [Infusion Event System](InfusionEventSystem.md) document for a full description of infusion events._
+
+The events fired by the Text To Speech component are described below.
+
+### `onStart` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                This event fires when the text has begun to be spoken.
```

Comment:
Do these start and stop events fire for each individual text in the queue? Or would there only be one stop event when the queue is finally emptied?